src/numJewelsInStones.js

```diff
@@ -29,6 +29,15 @@
  */
 function numJewelsInStones(jewels, stones) {
   // write code here
+  if (jewels === '') {
```

Comment:
Пустая строка в булевом значении `false`, равность можно не указывать
```suggestion
  if (!jewels) {
```